src/backend/graphItemService.ts

```diff
@@ -307,4 +317,30 @@ export class GraphItemService {
       badRows: [[RowErrorMessage.SOURCE_TARGET_NOT_FOUND, noExtremitiesRows]]
     };
   }
+
+  /**
+   * Ensure that the current user has the correct privileges to import nodes and create a report of the nodes imported.
+   *
+   * @param sourceKey
+   * @param rc
+   */
+  private static async assertHasReportNodeAccess(rc: RestClient, sourceKey: string): Promise<void> {
+    // The CSV_PLUGIN node will not be returned by LKE if:
+      // 1. As an admin, you are in strict schema and the category is not declared
+      // 2. As a custom group user, the category is not declared in the schema
```

Comment:
Is that mean all user of custom group have to create the category in the schema before using the csv plugin ?

Reply:
Yes, if they have the right to edit the schema

Reply:
To make it simple we can recommend that you are logged in as an admin and Linkurious is not in strict mode, wdyt ?